### ISSUE TITLE
Signal Cluster

### ISSUE THREAD

**Issue by just-nilux** [nateemma/strategies#33]
Hey, have you thought about using all the TradingSignals and then finding clusters of signals to build some kind of consensus for buys and sells? And eventually then also use the DWT signals as confirmation. Just an idea! Whats your take?

**Comment by tomjrtsmith** [comment]
Haven't done too much coding lately but had much the same thought when I
saw TradingSignals.py.

**Comment by just-nilux** [comment]
the function you have to find the best classifier could actually be base for that. Just a thought!

**Comment by nateemma** [comment]
I'm not quite sure what you mean.

The signals in TrainingSignals.py are all forward-looking, so I can't use
them in the models themselves, I can only use them to train the models. Did
you mean that I could combine various signals to create new 'compound'
signals for training (e.g. combine both adx and fbb signals)?

Thanks,

Phil

**Comment by just-nilux** [comment]
@nateemma 

> you mean that I could combine various signals to create new 'compound'
yes in a way. in a way like feature engineering, just in this case label engineering / selection. actually it boils down to that creating these sets and files manually like now is quite bad to maintain, if you want to make a lot of experiments its not practical and the best sets you may never discover.

are you using any pipeline to track the results, drift etc?
optuna with a pipeline to track results would be a perfect solution here I think!
